operations/patch_util.go

```diff
@@ -421,6 +429,12 @@ func gitCommitMessages(base, ref string) (string, error) {
 	return gitCmd("log", args...)
 }
 
+// assumes base includes @{upstream}
+func gitLastCommitMessage(base, ref string) (string, error) {
+	args := []string{"--no-show-signature", "--pretty=format:%B", fmt.Sprintf("%s...%s", base, ref), "-n 1"}
```

Comment:
Instead of `--pretty=format:%B`, I think you want `--pretty=format:%s`, since we want only the first line of the commit message.